index.bs

```diff
@@ -2418,7 +2418,7 @@ optionally evidence of [=user consent=] to a specific transaction.
 
     :   <dfn>attestation</dfn>
     ::  This member is intended for use by [=[RPS]=] that wish to express their preference for [=attestation conveyance=].
-        The default is {{AttestationConveyancePreference/none}}.
+        The value SHOULD be a member of {{AttestationConveyancePreference}} but [=client platforms=] MUST ignore unknown values, treating an unknown value as if the member was unset. The default is {{AttestationConveyancePreference/none}}.
```

Comment:
@equalsJeffH: The `default is...` text is unique to this entry, and in the actual generated doc quite duplicative since the default is shown. I was going to remove it from this sentence while I was here. Thoughts?

Reply:
ok